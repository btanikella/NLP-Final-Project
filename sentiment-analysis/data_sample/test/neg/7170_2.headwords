Grim
give
make
1/2
