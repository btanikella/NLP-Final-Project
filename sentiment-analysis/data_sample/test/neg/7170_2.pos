JJ RB IN JJ  JJ RB IN JJ  JJ RB IN JJ
PRP MD RB VB PRP  DT NNS   CC PRP RB MD VB PRP RP CC VB PRP TO VB
CC PRP MD RB VB PRP VB :  WP VBD PRP VBG
-LRB- SYM CD
