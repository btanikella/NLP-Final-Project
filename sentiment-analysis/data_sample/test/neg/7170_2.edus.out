Sentence #1 (15 tokens):
Grim instead of amusing, mean-spirited instead of playful, boring instead of interesting.
[Text=Grim CharacterOffsetBegin=4 CharacterOffsetEnd=8 PartOfSpeech=JJ] [Text=instead CharacterOffsetBegin=9 CharacterOffsetEnd=16 PartOfSpeech=RB] [Text=of CharacterOffsetBegin=17 CharacterOffsetEnd=19 PartOfSpeech=IN] [Text=amusing CharacterOffsetBegin=20 CharacterOffsetEnd=27 PartOfSpeech=JJ] [Text=, CharacterOffsetBegin=27 CharacterOffsetEnd=28 PartOfSpeech=,] [Text=mean-spirited CharacterOffsetBegin=29 CharacterOffsetEnd=42 PartOfSpeech=JJ] [Text=instead CharacterOffsetBegin=43 CharacterOffsetEnd=50 PartOfSpeech=RB] [Text=of CharacterOffsetBegin=51 CharacterOffsetEnd=53 PartOfSpeech=IN] [Text=playful CharacterOffsetBegin=54 CharacterOffsetEnd=61 PartOfSpeech=JJ] [Text=, CharacterOffsetBegin=61 CharacterOffsetEnd=62 PartOfSpeech=,] [Text=boring CharacterOffsetBegin=63 CharacterOffsetEnd=69 PartOfSpeech=JJ] [Text=instead CharacterOffsetBegin=70 CharacterOffsetEnd=77 PartOfSpeech=RB] [Text=of CharacterOffsetBegin=78 CharacterOffsetEnd=80 PartOfSpeech=IN] [Text=interesting CharacterOffsetBegin=81 CharacterOffsetEnd=92 PartOfSpeech=JJ] [Text=. CharacterOffsetBegin=92 CharacterOffsetEnd=93 PartOfSpeech=.] 
root(ROOT-0, Grim-1)
advmod(of-3, instead-2)
prep(Grim-1, of-3)
pobj(of-3, amusing-4)
punct(amusing-4, ,-5)
conj(amusing-4, mean-spirited-6)
advmod(of-8, instead-7)
prep(mean-spirited-6, of-8)
pobj(of-8, playful-9)
punct(playful-9, ,-10)
amod(playful-9, boring-11)
advmod(of-13, instead-12)
prep(boring-11, of-13)
pobj(of-13, interesting-14)
punct(Grim-1, .-15)

Sentence #2 (23 tokens):
It won't give you "the willies", but it just may gross you out or send you to sleep.
[Text=It CharacterOffsetBegin=98 CharacterOffsetEnd=100 PartOfSpeech=PRP] [Text=wo CharacterOffsetBegin=101 CharacterOffsetEnd=103 PartOfSpeech=MD] [Text=n't CharacterOffsetBegin=103 CharacterOffsetEnd=106 PartOfSpeech=RB] [Text=give CharacterOffsetBegin=107 CharacterOffsetEnd=111 PartOfSpeech=VB] [Text=you CharacterOffsetBegin=112 CharacterOffsetEnd=115 PartOfSpeech=PRP] [Text=`` CharacterOffsetBegin=116 CharacterOffsetEnd=117 PartOfSpeech=``] [Text=the CharacterOffsetBegin=117 CharacterOffsetEnd=120 PartOfSpeech=DT] [Text=willies CharacterOffsetBegin=121 CharacterOffsetEnd=128 PartOfSpeech=NNS] [Text='' CharacterOffsetBegin=128 CharacterOffsetEnd=129 PartOfSpeech=''] [Text=, CharacterOffsetBegin=129 CharacterOffsetEnd=130 PartOfSpeech=,] [Text=but CharacterOffsetBegin=131 CharacterOffsetEnd=134 PartOfSpeech=CC] [Text=it CharacterOffsetBegin=135 CharacterOffsetEnd=137 PartOfSpeech=PRP] [Text=just CharacterOffsetBegin=138 CharacterOffsetEnd=142 PartOfSpeech=RB] [Text=may CharacterOffsetBegin=143 CharacterOffsetEnd=146 PartOfSpeech=MD] [Text=gross CharacterOffsetBegin=147 CharacterOffsetEnd=152 PartOfSpeech=VB] [Text=you CharacterOffsetBegin=153 CharacterOffsetEnd=156 PartOfSpeech=PRP] [Text=out CharacterOffsetBegin=157 CharacterOffsetEnd=160 PartOfSpeech=RP] [Text=or CharacterOffsetBegin=161 CharacterOffsetEnd=163 PartOfSpeech=CC] [Text=send CharacterOffsetBegin=164 CharacterOffsetEnd=168 PartOfSpeech=VB] [Text=you CharacterOffsetBegin=169 CharacterOffsetEnd=172 PartOfSpeech=PRP] [Text=to CharacterOffsetBegin=173 CharacterOffsetEnd=175 PartOfSpeech=TO] [Text=sleep CharacterOffsetBegin=176 CharacterOffsetEnd=181 PartOfSpeech=VB] [Text=. CharacterOffsetBegin=181 CharacterOffsetEnd=182 PartOfSpeech=.] 
root(ROOT-0, give-4)
nsubj(give-4, It-1)
aux(give-4, wo-2)
neg(give-4, n't-3)
iobj(give-4, you-5)
punct(willies-8, ``-6)
det(willies-8, the-7)
dobj(give-4, willies-8)
punct(willies-8, ''-9)
punct(give-4, ,-10)
cc(give-4, but-11)
nsubj(gross-15, it-12)
advmod(gross-15, just-13)
aux(gross-15, may-14)
conj(give-4, gross-15)
dobj(gross-15, you-16)
prt(gross-15, out-17)
cc(gross-15, or-18)
conj(gross-15, send-19)
dobj(send-19, you-20)
aux(sleep-22, to-21)
xcomp(send-19, sleep-22)
punct(give-4, .-23)

Sentence #3 (15 tokens):
And it will certainly make you wonder: "what were they thinking?"
[Text=And CharacterOffsetBegin=187 CharacterOffsetEnd=190 PartOfSpeech=CC] [Text=it CharacterOffsetBegin=191 CharacterOffsetEnd=193 PartOfSpeech=PRP] [Text=will CharacterOffsetBegin=194 CharacterOffsetEnd=198 PartOfSpeech=MD] [Text=certainly CharacterOffsetBegin=199 CharacterOffsetEnd=208 PartOfSpeech=RB] [Text=make CharacterOffsetBegin=209 CharacterOffsetEnd=213 PartOfSpeech=VB] [Text=you CharacterOffsetBegin=214 CharacterOffsetEnd=217 PartOfSpeech=PRP] [Text=wonder CharacterOffsetBegin=218 CharacterOffsetEnd=224 PartOfSpeech=VB] [Text=: CharacterOffsetBegin=224 CharacterOffsetEnd=225 PartOfSpeech=:] [Text=`` CharacterOffsetBegin=226 CharacterOffsetEnd=227 PartOfSpeech=``] [Text=what CharacterOffsetBegin=227 CharacterOffsetEnd=231 PartOfSpeech=WP] [Text=were CharacterOffsetBegin=232 CharacterOffsetEnd=236 PartOfSpeech=VBD] [Text=they CharacterOffsetBegin=237 CharacterOffsetEnd=241 PartOfSpeech=PRP] [Text=thinking CharacterOffsetBegin=242 CharacterOffsetEnd=250 PartOfSpeech=VBG] [Text=? CharacterOffsetBegin=250 CharacterOffsetEnd=251 PartOfSpeech=.] [Text='' CharacterOffsetBegin=251 CharacterOffsetEnd=252 PartOfSpeech=''] 
root(ROOT-0, make-5)
cc(make-5, And-1)
nsubj(make-5, it-2)
aux(make-5, will-3)
advmod(make-5, certainly-4)
nsubj(wonder-7, you-6)
ccomp(make-5, wonder-7)
punct(make-5, :-8)
punct(make-5, ``-9)
nsubj(thinking-13, what-10)
aux(thinking-13, were-11)
nsubj(thinking-13, they-12)
ccomp(make-5, thinking-13)
punct(make-5, ?-14)
punct(make-5, ''-15)

Sentence #4 (4 tokens):
(*1/2)
[Text=-LRB- CharacterOffsetBegin=257 CharacterOffsetEnd=258 PartOfSpeech=-LRB-] [Text=* CharacterOffsetBegin=258 CharacterOffsetEnd=259 PartOfSpeech=SYM] [Text=1/2 CharacterOffsetBegin=259 CharacterOffsetEnd=262 PartOfSpeech=CD] [Text=-RRB- CharacterOffsetBegin=262 CharacterOffsetEnd=263 PartOfSpeech=-RRB-] 
root(ROOT-0, 1/2-3)
punct(1/2-3, -LRB--1)
dep(1/2-3, *-2)
punct(1/2-3, -RRB--4)

